llama_stack/apis/eval/eval.py

```diff
@@ -85,37 +112,42 @@ async def evaluate_rows(
         candidate: EvalCandidate,
     ) -> EvaluateResponse:
         """Evaluate a list of rows on a candidate.
-        
+
         :param dataset_rows: The rows to evaluate.
         :param scoring_fn_ids: The scoring function ids to use for the evaluation.
         :param candidate: The candidate to evaluate on.
         :return: EvaluateResponse object containing generations and scores
         """
 
-    @webmethod(route="/eval/benchmarks/{benchmark_id}/jobs/{job_id}", method="GET")
-    async def job_status(self, benchmark_id: str, job_id: str) -> Optional[JobStatus]:
-        """Get the status of a job.
+    @webmethod(route="/eval/jobs", method="GET")
+    async def list_eval_jobs(self) -> List[EvalJob]:
+        """List all evaluation jobs.
 
-        :param benchmark_id: The ID of the benchmark to run the evaluation on.
-        :param job_id: The ID of the job to get the status of.
-        :return: The status of the evaluationjob.
+        :return: A list of evaluation jobs.
         """
         ...
 
-    @webmethod(route="/eval/benchmarks/{benchmark_id}/jobs/{job_id}", method="DELETE")
-    async def job_cancel(self, benchmark_id: str, job_id: str) -> None:
-        """Cancel a job.
+    @webmethod(route="/eval/jobs/{job_id}", method="GET")
+    async def get_eval_job(self, job_id: str) -> Optional[EvalJob]:
+        """Get a job by id.
 
-        :param benchmark_id: The ID of the benchmark to run the evaluation on.
-        :param job_id: The ID of the job to cancel.
+        :param job_id: The id of the job to get.
+        :return: The job.
         """
         ...
 
-    @webmethod(route="/eval/benchmarks/{benchmark_id}/jobs/{job_id}/result", method="GET")
-    async def job_result(self, benchmark_id: str, job_id: str) -> EvaluateResponse:
-        """Get the result of a job.
+    @webmethod(route="/eval/jobs/{job_id}", method="DELETE")
+    async def delete_eval_job(self, job_id: str) -> Optional[EvalJob]:
+        """Delete a job.
 
-        :param benchmark_id: The ID of the benchmark to run the evaluation on.
-        :param job_id: The ID of the job to get the result of.
-        :return: The result of the job.
+        :param job_id: The id of the job to delete.
         """
+        ...
+
+    @webmethod(route="/eval/jobs/{job_id}/cancel", method="POST")
+    async def cancel_eval_job(self, job_id: str) -> Optional[EvalJob]:
```

Comment:
same, I'd expect it to return `nil` on success.

Reply:
this could also be `PUT .../jobs/{job_id}` w/ `{"status" = "cancelled"}`

Reply:
Interesting. And `pause` would be `status = paused` etc.?

Reply:
could be, if there's a JobStatus.paused.

an argument against this is that clients can only trigger a subset of transitions, e.g. -> cancelled but not -> scheduled / completed, and those transitions become paths themselves.